fighters/common/src/general_statuses/run.rs

```diff
@@ -162,15 +167,25 @@ unsafe fn status_runbrake_main(fighter: &mut L2CFighterCommon) -> L2CValue {
     symbol = "_ZN7lua2cpp16L2CFighterCommon19status_TurnRunBrakeEv")]
 unsafe fn status_turnrunbrake(fighter: &mut L2CFighterCommon) -> L2CValue {
     fighter.status_TurnRunBrake_Sub();
-
     let dash_hip_offset_x = VarModule::get_float(fighter.battle_object, vars::common::instance::DASH_HIP_OFFSET_X);
     let run_hip_offset_x = VarModule::get_float(fighter.battle_object, vars::common::instance::RUN_HIP_OFFSET_X);
     let mut hip_translate = Vector3f::zero();
     MotionModule::joint_local_tra(fighter.module_accessor, Hash40::new("hip"), false, &mut hip_translate);
     hip_translate.z += (dash_hip_offset_x - run_hip_offset_x) * 0.5;
     ModelModule::set_joint_translate(fighter.module_accessor, Hash40::new("hip"), &Vector3f{ x: hip_translate.x, y: hip_translate.y, z: hip_translate.z }, false, false);
     
-    fighter.sub_shift_status_main(L2CValue::Ptr(L2CFighterCommon_bind_address_call_status_TurnRunBrake_Main as *const () as _))
+    fighter.sub_shift_status_main(L2CValue::Ptr(status_turnrunbrake_main as *const () as _))
```

Comment:
Please call `fighter.main_shift(status_turnrunbrake_main)` for type safety guarantees and readability (we've had this for ages)

Reply:
Done